src/brushes/single-sphere.js

```diff
@@ -0,0 +1,29 @@
+/* globals AFRAME THREE */
+AFRAME.registerBrush('single-sphere',
+  {
+    init: function (color, width) {
+      this.material = new THREE.MeshStandardMaterial({
+        color: this.data.color,
+        roughness: 0.6,
+        metalness: 0.2,
+        side: THREE.FrontSide,
+        shading: THREE.SmoothShading
+      });
+      this.geometry = new THREE.IcosahedronGeometry(0.5, 2);
```

Comment:
Use`IcosahedronBufferGeometry`. It's always better to use the `BufferGeometry` version of each primitive.